src/components/product-card/product-card.component.html

```diff
@@ -0,0 +1,50 @@
+<section> <!-- I have a section tag here, however, I feel that it may need to be removed if we dynamically add multiple of these components-->
+    <div class="container py-5">
+      <div>
+        <div class="col-md-8 col-lg-6 col-xl-4">
+          <div class="card">
+            <img src="/assets/spongebobChair.jpg"
+              class="card-img-top" alt="spongebob chair" />
+            <div class="card-body">
+              <div class="text-center">
+                <h5 class="card-title">ITEM NAME</h5>
```

Comment:
Instead of hardcoding "ITEM NAME", could we maybe use `@Input` to make this a component attribute? https://angular.io/api/core/Input Can also look at the [title component](https://github.com/abellnoel/nooblion-commerce/tree/main/src/components/title) for an example.

Reply:
Same for description. I think this would be fine to do now since those are pretty universal parts of a product. Maybe even price.